api/controllers/arithmeticController.js

```diff
@@ -11,7 +11,7 @@ exports.calculate = function(req, res) {
   });
 
   var operations = {
-    'add':      function(a,b) { return +a + +b },
```

Comment:
Please provide the + sign before the variables. This indicates to the JS engine to perform arithmetic operation.